firmware/quadruna/VC/src/tasks.c

```diff
@@ -562,7 +556,6 @@ _Noreturn void tasks_runCanRx(void)
         CanMsg rx_msg;
         io_can_popRxMsgFromQueue(&rx_msg);
         io_telemMessage_pushMsgtoQueue(&rx_msg);
-        io_canLogging_pushTxMsgToQueue(&rx_msg);
```

Comment:
Same with this

Reply:
same with here too. should have fixed these in other PR but was feeling lazy